docs/apv.md

```diff
@@ -511,6 +511,17 @@ Overtreding van lid 1 wordt bestraft met een straf in de 1e categorie.
 4. Een uitzondering op lid 2 is narcopolis hier op mogen gangs niet samenwerken.
 5. Bij overtreding van lid 1 en 3. Zal dit resulteren in een straf volgens de 5e categorie voor iedereen die meedeed aan het Roleplayscenario. Bij herhaling na de straf van de 5e categorie wordt er een gang strike uitgedeeld.
 
+### Artikel 55 - Appartementen
+
+1. Het uitvoeren van activiteiten in je appartement is verboden en valt onder het misbruiken maken van je appartement.
```

Comment:
dit is wel heel breed mag ik niet adem halen? dat is namelijk ook een activiteit

Reply:
whoops